dynawo/3rdParty/CMakeLists.txt

```diff
@@ -0,0 +1,617 @@
+# Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+# See AUTHORS.txt
+# All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+
+# Minimum required (for ExternalProject_Add)
+cmake_minimum_required(VERSION 3.9.6 FATAL_ERROR)
+
+option(BUILD_SHARED_LIBS "Build shared libraries (.dll/.so) instead of static ones (.lib/.a)" ON)
+set(CMAKE_BUILD_TYPE "Release" CACHE STRING "Choose the type of build, options are: Debug Release RelWithDebInfo MinSizeRel")
+option(CXX11_ENABLED "Enable support of C++11 standard" TRUE)
+set(MSVC_STATIC_RUNTIME_LIBRARY OFF CACHE BOOL "Choose either ON or OFF.")
+
+if(NOT DEFINED OPENMODELICA_INSTALL)
+  message(FATAL_ERROR "You must define OPENMODELICA_INSTALL variable.")
+endif()
+set(OPENMODELICA_INSTALL CACHE PATH "Path where OpenModelica (patched for Dynawo) will be installed")
+
+if(NOT DEFINED OPENMODELICA_SRC)
+  message(FATAL_ERROR "You must define OPENMODELICA_SRC variable.")
+endif()
+set(OPENMODELICA_SRC CACHE PATH "Path where OpenModelica sources will be cloned")
+
+if(POLICY CMP0074)
```

Comment:
Please add a comment to explain in which condition this is used.

Reply:
it is only there to disable warnings and looking in cmake doc gives all the details

Reply:
When is it used in Dynawo compilation?

Reply:
Cmake warnings not compilation warnings. It is only a cmake thing no relation to Dynawo